test/unit/app/storage.test.js

```diff
@@ -0,0 +1,55 @@
+const { BlobServiceClient } = require('@azure/storage-blob')
+const { getBlob, resetClient } = require('../../../app/storage')
+const { streamToBuffer } = require('../../../app/lib/streamToBuffer')
+const { storage } = require('../../../app/config')
+
+jest.mock('@azure/storage-blob')
```

Comment:
this is already mocked so you don't need to do it again here

Reply:
https://github.com/DEFRA/ffc-ahwr-dashboard/blob/main/__mocks__/%40azure/storage-blob.js